docs/Helix.md

```diff
@@ -120,3 +120,16 @@ There are two main ways to opt out of helix
 - Skipping an individual test via `[SkipOnHelix("url to github issue")]`.
 
 Make sure to file an issue for any skipped tests and include that in a comment next to either of these
+
+## Process for updating helix matrix
+
+Goal is to balance cost/flakiness against having some coverage of supported distros:
+- At the start of each product version, we pick a set of queues/versions/arches to run based on popularity and perceived risk, and how long is left in the support for that OS version.
+- Whenever a new OS is coming online, we ask CTI to do a run on it, and we submit a PR to add a queue for it to check for any failures in it, but if there aren’t any, we don’t merge it.
+- Link to [OS support calendar] (https://dev.azure.com/devdiv/DevDiv/_wiki/wikis/DevDiv.wiki/12624/OS-Version-Management-Calendar-2021)
+- Link to [current list of queues](../eng/targets/Helix.Common.props)
+
+## Example of adding a new docker image to helix
+
+- Example PR: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/398
```

Comment:
Do you also have an example dotnet/aspnetcore PR using the queue added in dotnet/dotnet-buildtools-prereqs-docker#398❔

nit (not sure why GitHub doesn't recognize the URI):
```suggestion
- Example PR: dotnet/dotnet-buildtools-prereqs-docker#398
```

Reply:
That PR after will come once the queue is up and healthy, as we'll need to switch fedora over to it, I can link that PR to the docs as part of the PR